ui/src/reducers/__tests__/index.test.js

```diff
@@ -3,46 +3,61 @@ import { APPS_SUCCESS, APPS_SORT, APPS_FAILURE, TOGGLE_HEADER_DROPDOWN } from '.
 import { SortByDirection, sortable } from '@patternfly/react-table';
 
 describe('reducer', () => {
-  const columns = [
-    { title: 'App Name', transforms: [ sortable ] },
-    { title: 'App ID', transforms: [ sortable ] },
-    { title: 'Deployed Versions', transforms: [ sortable ] },
-    { title: 'Current Installs', transforms: [ sortable ] },
-    { title: 'Launches', transforms: [ sortable ] }
+  const initialState = {
+    apps: { rows: [], data: {} },
+    sortBy: { direction: SortByDirection.asc, index: 0 },
+    columns: [
+      { title: 'App Name', transforms: [ sortable ] },
+      { title: 'App ID', transforms: [ sortable ] },
+      { title: 'Deployed Versions', transforms: [ sortable ] },
+      { title: 'Current Installs', transforms: [ sortable ] },
+      { title: 'Launches', transforms: [ sortable ] }
+    ],
+    appDetailRows: [
```

Comment:
@laurafitzgerald @craicoverflow going to leave this test data in place until PR by @craicoverflow to retrieve data from API call is in place. Can remove now if preferred, but may make testing more difficult until API is pulling real data